src/Http/Routing/src/ParameterPolicyActivator.cs

```diff
@@ -1,8 +1,6 @@
 // Licensed to the .NET Foundation under one or more agreements.
 // The .NET Foundation licenses this file to you under the MIT license.
 
-#nullable disable
```

Comment:
Note: Nothing has functionally changed in this class. I made changes to it (enable nullable, split up methods) intending to use it, but found I didn't need to. I left the changes here as they're all positive.